src/common/options.cc

```diff
@@ -3791,6 +3791,10 @@ std::vector<Option> get_global_options() {
     .set_default(false)
     .set_description(""),
 
+    Option("rocksdb_max_items_rmrange", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(1024)
+    .set_description(""),
```

Comment:
could you add `.see_also("rocksdb_enable_rmrange")` ? and add a brief description?